uint16 device
time ts
uint8[15000] buf
uint32 size